packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -222,28 +224,38 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
   Widget _expandCollapseButtons() {
     if (!_expandCollapseSupported) return null;
 
+    const minIncludeTextWidth = 900.0;
+    final buttonStyle = denseAwareOutlinedButtonStyle(
+      context,
+      minIncludeTextWidth,
+    );
+
     return Align(
       alignment: Alignment.centerRight,
       child: Row(
         mainAxisAlignment: MainAxisAlignment.end,
         mainAxisSize: MainAxisSize.min,
         children: [
-          Flexible(
+          SizedBox(
             child: OutlinedButton(
```

Comment:
Use the `IconLabelButton` class for the Expand and Collapse buttons, then you can get rid of the `denseAwareOutlinedButtonStyle` code here since it is already used in IconLabelButton